src/components/PersonLink.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  name?: string,
+  hasParent?: Person,
+};
+
+export const PersonLink: React.FC<Props> = ({
+  name,
+  hasParent,
+}) => {
+  return (
+    <td>
+      {hasParent ? (
+        <Link
+          to={`/people/${hasParent.slug}`}
+          className={classNames(
+            { 'has-text-danger': hasParent.sex === 'f' },
+          )}
+        >
+          {hasParent.name}
+        </Link>
+      ) : (
+        name
+      )}
+    </td>
```

Comment:
you don't need one more `td` here. it causes additional line under the name
![image](https://user-images.githubusercontent.com/22824431/221569110-1cc08ded-03a8-4e6c-9848-21f6a0c54372.png)